src/List/ItemList.js

```diff
@@ -38,9 +59,13 @@ const Items = props => {
         </Fragment>
       ) : (
         <ul>
-          {dbItems.map((item, index) => {
-            return <li key={index}>{item.name}</li>;
-          })}
+          {dbItems
+            .filter(item =>
+              item.name.toLowerCase().includes(filterInput.toLowerCase())
```

Comment:
I would like to see you extract this out to make it a bit more readable, ie putting the toLowercases on another line so that someone can read this function and see exactly what it's doing instead of pre-filtering logic.

Reply:
I updated the file to put them on new lines but when pushing the commit, it auto puts it in one line... is there a setting or is this expected? @rmorabia

Reply:
I guess I mean I want y'all to extract this line into it's own function so you can make readable variables for what touches .toLowerCase().